dispatcher/ws/wsHandler.go

```diff
@@ -0,0 +1,71 @@
+package ws
+
+import (
+	"net/http"
+
+	"github.com/ElrondNetwork/elrond-go-logger/check"
+	"github.com/ElrondNetwork/notifier-go/dispatcher"
+)
+
+// ArgsWebSocketHandler defines the argument needed to create a websocketHandler
+type ArgsWebSocketHandler struct {
+	Hub      dispatcher.Hub
+	Upgrader dispatcher.WSUpgrader
+}
+
+type websocketHandler struct {
```

Comment:
can we rename it? we usually name interfaces with a `handler` suffix

Reply:
renamed with `processor` suffix